src/payment/strategies/moneris/moneris.ts

```diff
@@ -1,9 +1,43 @@
+/**
+ * A set of stringified CSS to apply to Moneris' IFrame fields.
+ * CSS attributes should be converted to string.
+ * Please note that ClassNames are not supported.
+ *
+ * IE:
+ * ```js
+ * {
+ *      cssBody: 'background:white;';
+ *      cssTextbox: 'border-width:2px;';
+ *      cssTextboxPan: 'width:140px;';
+ *      cssTextboxExpiry: 'width:40px;';
+ *      csstexboxCvd: 'width:40px;';
+ * }
+ * ```
+ *
+ * When using several attributes use semicolon to separate each one.
+ * IE: 'background:white;width:40px;'
+ */
 export default interface MonerisStylingProps {
+    /**
+     * Stringified CSS to apply to the body of the IFrame.
+     */
     cssBody?: string;
+    /**
+     * Stringified CSS to apply to each of input fields.
+     */
     cssTextbox?: string;
+    /**
+     * Stringified CSS to apply to the card's number field.
+     */
     cssTextboxPan?: string;
+    /**
+     * Stringified CSS to apply to the card's expiry field.
+     */
     cssTextboxExpiry?: string;
-    csstexboxCvd?: string;
+    /**
+     * Stringified CSS to apply to the card's CVV field.
+     */
+    cssTexboxCvd?: string;
```

Comment:
should this be `cssTextboxCvd` ?

and why do we use `CVV` in the description but `Cvd` in the attribute name?

Reply:
CVV is the word that BigCommerce always use for this field but cvd is the way that Moneris use

Reply:
(I was originally pointing out the missing `t` in `cssTex_boxCvd`)

It seems we're mapping our field names to Moneris field names here: https://github.com/bigcommerce/checkout-sdk-js/pull/1172/files#diff-d755dc9987b2df0ced566e63258108d7af6892bc592ed1cccd9f4ddbe119dfc9R135

Where we're just going from Moneris snake_case to camelCase.

So if BC users are accustomed to `CVV` over `CVD` and `Card Number` over `PAN`, could we not do something like:

`
 css_textbox_pan: style?.cardNumber || 'width:140px;',
 css_textbox_cvd: style?.cvv || 'width:40px',
`
(maybe there's a better precedent already in our repo we could follow)

That way they they won't need the comments to help them map BC to Moneris terminology.